integration/unplugin/src/index.ts

```diff
@@ -13,49 +13,31 @@ import {
 } from '@danielx/civet/ts-diagnostic';
 import * as fs from 'fs';
 import path from 'path';
-import ts from 'typescript';
+import type { FormatDiagnosticsHost, Diagnostic, System } from 'typescript';
 import * as tsvfs from '@typescript/vfs';
 import type { UserConfig } from 'vite';
 import os from 'os';
 
-const formatHost: ts.FormatDiagnosticsHost = {
-  getCurrentDirectory: () => ts.sys.getCurrentDirectory(),
-  getNewLine: () => ts.sys.newLine,
-  getCanonicalFileName: ts.sys.useCaseSensitiveFileNames
-    ? f => f
-    : f => f.toLowerCase(),
-};
-
 export type PluginOptions = {
```

Comment:
I think it's good if the old `dts: boolean, js: boolean` is kept for a little bit, deprecated with a warning of course. `dts` would map plainly to `emitDeclaration`. `js: true` can map to `ts: "civet"`

Reply:
Sounds good to me; it's really easy to still support them.

I like offering `js: true` maybe even long-term, because it corresponds to an option to the Civet compiler. (Someone was recently wondering why `inlineMap: true` didn't work. I'm not sure it's really useful for an unplugin, but it points toward mirroring Civet's compiler options.)

I think we could remove these options from the types, though; this is probably sufficient warning that there's a better way now.